docs/How-To/cli/README.md

```diff
@@ -4,33 +4,33 @@ Description: A high-level overview of how the command-line interface (CLI) works
 ---
 # CLI Reference
 
-This document provides a high-level overview of how the command line (CLI) interface works for the fundraising module. The executable file name is called `fundraisingd`.
+## Synopsis
 
-## Command Line Interface
+This document provides a high-level overview of how the command line (CLI) interface works for the `fundraising` module. To set up a local testing environment, it requires the latest [Starport](https://starport.com/). If you don't have Starport set up in your local machine, see [this Starport guide](https://docs.starport.network/) to install it. Run this command under the project root directory `$ starport chain serve -c config-test.yml` or simply `$ make localnet`.
+
+Note that [jq](https://stedolan.github.io/jq/) is recommended to be installed as it is used to process JSON throughout the document.
 
-To test out the following commands, you must set up a local network. By simply running `$ make localnet` under the root project directory, you can start the local network. It requires the latest [Starport](https://starport.com/). If you don't have `Starport` set up in your local machine, see this [Starport guide](https://docs.starport.network/) to install it.  
+## Command Line Interface
 
-- [CLI Reference](#cli-reference)
-  - [Command Line Interface](#command-line-interface)
-  - [Transaction](#transaction)
-    - [CreateFixedPriceAuction](#createfixedpriceauction)
-    - [CreateBatchAuction](#createbatchauction)
-    - [CancelAuction](#cancelauction)
-    - [AddAllowedBidder](#addallowedbidder)
-    - [PlaceBid](#placebid)
-    - [ModifyBid](#modifybid)
-  - [Query](#query)
-    - [Params](#params)
-    - [Auctions](#auctions)
-    - [Auction](#auction)
-    - [Bids](#bids)
-    - [Vestings](#vestings)
+- [Transaction](#transaction)
+  - [CreateFixedPriceAuction](#createfixedpriceauction)
+  - [CreateBatchAuction](#createbatchauction)
+  - [CancelAuction](#cancelauction)
+  - [AddAllowedBidder](#addallowedbidder)
```

Comment:
We defined AddAlloedBidder that can be called from the outside of the module. Is there any difference from the other modules?

Reply:
`AddAllowedBidder` CLI command is only available for the testing purpose. It becomes available when `fundraisingd` is built with `config-test.yml` file which passes custom ldflags to keeper. Without `AddAllowedBidders` CLI command, there is currently no way for developers to test out `PlaceBid` or `ModifyBid`.